packages/gatsby/src/internal-plugins/functions/gatsby-node.ts

```diff
@@ -220,7 +220,7 @@ export async function onPreBootstrap({
       function callback(err, stats): any {
         const rawMessages = stats.toJson({ moduleTrace: false })
         if (rawMessages.warnings.length > 0) {
-          reporter.warn(reportWebpackWarnings(rawMessages.warnings))
+          reporter.warn(reportWebpackWarnings(rawMessages.warnings, reporter))
```

Comment:
Isn't this the correct function blueprint? Doesn't reportWebpackWarnings also log it?
```suggestion
          reportWebpackWarnings(rawMessages.warnings, reporter)
```